SnM/SnM_Misc.cpp

```diff
@@ -217,6 +217,24 @@ bool SNM_SetDoubleConfigVar(const char *varName, double newValue) {
 	return ConfigVar<double>(varName).try_set(newValue);
 }
 
+bool SNM_GetLongConfigVar(const char *varName, int *highOut, int *lowOut) {
+	auto var = ConfigVar<long long>(varName);
+	if (!var)
+		return false;
+
+	if (highOut)
+		*highOut = *var >> 32;
+	if (lowOut)
+		*lowOut = static_cast<int>(*var);
```

Comment:
Nitpick, but how about
`*lowOut = static_cast<int32_t>(*var);`?

Likely doesn't affect anything in practice but would formally more correct since `int` isn't (formally) guaranteed to be 32 bit, no? :)

Reply:
Maybe (in fact in the original version of this patch I used `int64_t` instead of `long long`, but I changed my mind to match with the `int` arguments).

If `int` were to change size in the (far) future, `highOut` wouldn't be valid anymore, so you might have a point. In this case I should also do `newLowValue & 0xFFFFFFFF` below.

Or use `sizeof(int)*8` in both functions instead of hard-coding 32. Which one is correct depends on whether configvar use fixed-size integers in REAPER's source...

Reply:
Thanks.
Actually this wasn't meant to appear as a proper 'review', rather just a comment/thought, so I'll mark it as resolved.